ExtensionHandler/Linux/src/tests/test_HandlerUtil.py

```diff
@@ -118,6 +120,15 @@ def test_file_fallback(self, _urllib_mock):
             # while trying to open ../net6.json
             self.assertEqual(open_mock.call_count, 2)
 
+class TestNet6FileConsistency(unittest.TestCase):
+    def test_file_consistency(self):
+        with open("net6.json", encoding="utf-8") as net6_file:
+            local = json.loads(net6_file.read())
+        remote_net6_fileurl = dedent('''
```

Comment:
you have nothing to dedent here :) Just use the string directly.

Reply:
The Pylint was saying that the line is too long if I put the entire string in one place without using dedent.

Reply:
Okay I believe I can remove the warning also by using # pylint: disable=line-too-long. Will make the changes.